dist/backdraft.js

```diff
@@ -1214,42 +1257,67 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
+      window.activeMenu = null;
       var filter = col.filter;
+      var filterActive = filter.value || filter.eq || filter.gt || filter.lt;
       // create filtering wrapper div
       var wrapperDiv = document.createElement('div');
-      wrapperDiv.className = "DataTables_filter_wrapper";
+      wrapperDiv.className = "dropdown DataTables_filter_wrapper";
       wrapperDiv.id = "wrapper-" + col.attr;
-      wrapperDiv.innerHTML = "Filter";
+      wrapperDiv.innerHTML = this.filterButtonTemplate({ active: filterActive });
 
       // determine how many columns we need if we're dealing with list filtering
       var listClass = "";
       if (filter.type === "list") {
-        if (filter.options.length > 30)
-          listClass = " triple"
-        else if (filter.options.length > 15)
-          listClass = " double"
-        else
+        if (filter.options.length > 30) {
+          listClass = " triple";
+        } else if (filter.options.length > 15) {
+          listClass = " double";
+        } else {
           listClass = " single";
+        }
       }
 
       // create filtering menu div
       var filterDiv = document.createElement("div");
-      filterDiv.className = "filterMenu" + listClass;
+      filterDiv.className = "filterMenu dropdown-menu" + listClass;
       filterDiv.id = "menu-" + col.attr;
-      wrapperDiv.appendChild(filterDiv);
 
       // put filtering controls in filter div and put wrapper div in header
       if (filter.type === "string")
         $('input', head).appendTo(filterDiv)
       else
         $('ul', head).appendTo(filterDiv);
+      $('button', head).appendTo(filterDiv);
+      wrapperDiv.appendChild(filterDiv);
       head.appendChild(wrapperDiv);
 
-      // handle hovering on wrapperDiv
-      $('.DataTables_filter_wrapper', head).hover(function () {
-        $('.filterMenu', head).slideDown(200);
-      }, function () {
-        $('.filterMenu', head).slideUp(50);
+      // handle filter menu display
```

Comment:
well "Clear" isn't a cancel button, it removes the active filters.  

* Both "Clear" and "Filter" should close the menu
* Opening a new menu should close the menu
* Clicking on the toggle button should toggle the menu
* Clicking out of the menu should close it.
** Else the user has to click filter, clear, or the toggle button.  This doesn't cover the use case where the user wants to apply filters to multiple columns before submitting the filter request to the server and isn't aware they can click the toggle button to make the menu disappear, which isn't always the case.